### ISSUE TITLE
Is Local Command Execution supported?

### ISSUE THREAD

**Issue by neidiom** [braintree/runbook#52]
In the docs everything is executed via Remote Command Execution/SSH but is there a way to do execution on the localhost?

**Comment by pblesi** [comment]
Yes. There are a few different ways to perform local command execution. The first is to set the target server for [SSH command](https://github.com/braintree/runbook#command) to be `localhost`/`127.0.0.1`. The second is to use the [tmux_command](https://github.com/braintree/runbook#command) to execute the command in a local tmux pane (See also [Tmux Layouts](https://github.com/braintree/runbook#tmux-layouts). The third way is to use [ruby_command](https://github.com/braintree/runbook#ruby-command) to execute a command using custom Ruby.
